lib/elastic_record/index/settings.rb

```diff
@@ -18,7 +18,8 @@ def settings
           end
       end
 
-      def update_settings(index_name = alias_name)
+      def update_settings(index_name = alias_name, settings: self.settings)
+        self.settings = settings
```

Comment:
erm... Calling the method changes the state of the object?

Reply:
Yes - see the code above. It calls `deep_merge!` so that multiple calls to `elastic_index.settings=` work, I guess.

Reply:
Right, but this is really bad. The index instance is a global object, and `update_settings` can be called through the lifetime of an app....

Reply:
For the settings I have in mind, it's harmless. But I could have it not do this and then if you pass in settings, those are the ones that are updated. Right now the alternative is that the only way for an app to do a partial settings update is to call `json_put /{index}/_settings` directly.

Reply:
Fixed and updated the PR description.